app.js

```diff
@@ -6,28 +6,26 @@ const { loadDatabase, searchDatabase } = require('./googleSpreadsheet');
 
 const app = express();
 
-const allScans = [];
+const allScans = new Map();
 
```

Comment:
why?

Reply:
Because I want to avoid repetitions in the "recent" page.
I want to add one by one unique items on that page

Reply:
This is not the way to handle changes and features. We create an issue, discuss it and push it.
Hidding secret features in unrelated commits is not appropriate decision workflow and distracting for code review